packages/shared/locales/en.json

```diff
@@ -748,11 +748,11 @@
             "unstakedSuccessfully": "Your funds have been unstaked for {account}."
         },
         "stakingConfirmation": {
-            "title": "Airdrop Participation",
+            "title": "Staking Confirmation",
```

Comment:
It might be better to use "Confirm stake" here to be more consistent with most other popups we have. We should also probably change the "Staking Manager" title to "Manage stake" as well.

Reply:
Changed it